k8s/configMap-sjc.yaml

```diff
@@ -33,3 +33,6 @@ data:
       - Id: services-all
         FrontendRegexp: ^[^\.]+\.k8s\.wikia\.net
         SamplingRate: 1.0
+      - Id: helios
```

Comment:
I would prefer to add specific rules:
* helios
* pandora (maybe even specific ones for each service)

The first two `generic-*` were done to test if the rules work correctly and should disappear when we add service specific ones.